README.md

```diff
@@ -61,8 +61,6 @@ PUBLIC_TRANSPORT
 WALKING_FERRY
 CYCLING_FERRY
 DRIVING
-WALKING
```

Comment:
As discussed via DM, not sure you need to remove these - they should work on prod.